_positions/product-manager.md

```diff
@@ -0,0 +1,115 @@
+- Establish the bar for quality and the definition of done.
+- Identify and proactively address project risks.
+- Explore new tools, methodologies, processes, etc. and share lessons with other product leads.
+- Mentor others, give constructive criticism, and improve practices and standards on your teams.
+- Actively engage with the team at large — take part in cross-project and cross-domain conversation, skill- and knowledge-share, and working groups or guilds.
+
+**Objective #3: You will manage relationships with 18F’s agency partners.**<br>
+- Build trusted relationships with agency partners and help them understand 18F’s user centered, iterative approach to understanding the problem and building a solution.
+- Reinforce 18F’s “way we work” values (building in the open, using open source, etc.).
+- Make sure agency partners are fully engaged members of the team and help them manage expectations with their stakeholders.
+- Give agency partners advice they need to hear instead of what they want to hear. Promote best practices by making a strong case even in the face of resistance.
+- Ensure projects are delivered on budget and within the terms of our agreements. Define and execute plans to successfully transition projects at the end of 18F’s engagements to achieve lasting results.
+- Evaluate potential work for impact, risks, and alignment with 18F’s values.
+
+
+##Qualifications:
+Qualifications are based on length and level of experience.  Therefore, in addition to describing duties performed, applicants must provide the exact dates of each period of employment (from month/year to month/year) and the number of hours per week if part time.  Qualification determinations cannot be made when resumes do not include the required information, so failure to provide this information may result in disqualification.
+
+**Applicants applying for this job must meet the following requirements:**
+
+Have product management related experience demonstrating *each* of the five competencies *and* specialized experience listed below:
+- **Communication**: Ability to communicate effectively with a variety of audiences to establish a shared vision and understanding of the project’s objectives and goals. 
+- **Product Experience**: Ability to deliver the right product using user-centered and agile-based methodologies.
+- **Leadership**: Ability to effectively lead a cross-functional team to meet project/product objectives. 
+- **Innovation**: Ability to apply innovative tools and technologies  and to help partners make strategic & tactical decisions toward innovation. 
+- **18F Core Values Alignment**: The ability to work with integrity, transparency and resiliency in civic minded or high impact environment. 
+
+###Specialized Experience:
+**GS-14**
+To qualify at a GS-14 level you must demonstrate at least one year of specialized experience at the next lower grade level GS-13, or equivalent in the federal service:
```

Comment:
The info on that page is not the most relevant to this role because it refers to what the Innovation Specialist looks like at GS13-15, based off the Innovation Specialist PD. We can link to the "govt pay grades explained" as a reference but I'd want to note clearly that it is only for reference.